apps/src/applab/gridUtils.js

```diff
@@ -28,8 +28,8 @@ export function scaledDropPoint(draggedElement) {
   const xScale = boundingRect.width / div.offsetWidth;
   const yScale = boundingRect.height / div.offsetHeight;
 
-  let left = (draggedOffset.left - $(div).offset().left) / xScale;
-  let top = (draggedOffset.top - $(div).offset().top) / yScale;
+  let left = Math.round((draggedOffset.left - $(div).offset().left) / xScale);
+  let top = Math.round((draggedOffset.top - $(div).offset().top) / yScale);
```

Comment:
What was the purpose for this change?

Reply:
This was an existing issue where we were setting floating point values, which caused a bug when the values were incorrectly parsed by other code. I found this while debugging something else and decided to fix it.

Reply:
👍